src/app/products/loan-products/models/loan-product.model.ts

```diff
@@ -125,6 +131,7 @@ export interface LoanProduct {
   feeToIncomeAccountMappings?:                               ChargeToIncomeAccountMapping[];
   penaltyToIncomeAccountMappings?:                           ChargeToIncomeAccountMapping[];
   enableAccrualActivityPosting?:                             boolean;
+  supportedInterestRefundTypes?:                             StringEnumOptionData[];
```

Comment:
Use the `OptionData` already used here in this interface

Reply:
Its not the same. OptionData id is number. StringEnumOptionData id is string